.github/workflows/blogs-indexing.yaml

```diff
@@ -0,0 +1,46 @@
+name: Indexing blogs
+on:
+  push:
+    branches: ['search-blogs-ci-indexing']
+    paths:
+      - 'content/blog/**'
+jobs:
+  build-and-indexing-blogs:
+    name: Build and indexing blogs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js and use yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
```

Comment:
Should we update this to `16.x` since that's the latest node engine that dvc.org can use?

Reply:
Sure, we can use that.

Reply:
Ended up using `node@14`. It looks like we have some issue with `node@16`. 
It seems like the issue is with `gatsby-plugin-manifest`. Need to look into detail. 

<img width="1604" alt="Screen Shot 2022-02-11 at 19 52 28" src="https://user-images.githubusercontent.com/20840228/153610957-3eee263f-0f2b-48f0-a769-9de0062ca080.png">